app/controllers/game_controller.rb

```diff
@@ -0,0 +1,15 @@
+class GameController < ApplicationController
+
+  #history of all games
+  def history
+    per_page = params[:per_page].try(:to_i) || 10
+    page = params[:page].try(:to_i) || 1
+    games = Game.order(updated_at: :desc).limit(per_page).offset((page - 1) * per_page)
```

Comment:
The `created_at` date is a better column to sort on to get the order in which the games took place. 
Also, it would be a good idea to have a secondary sort on `id` just in case the `created_at` times are the same. (Unlikely, but not hard to add a secondary sort just in case).
Also, if we add the `will_paginate` gem to the Gemfile, we can use `paginate` here to do the paging for us.
```suggestion
    games = Game.order(created_at: :desc, id: :desc).paginate(page: page, per_page: per_page)
```